js/app.js

```diff
@@ -0,0 +1,101 @@
+// Creando función cipher
+
+function cipher(str) {
+  var index1 = []; // [65, 66, 67] || [97, 98, 99]
+  var indexAscii = []; // [72, 73, 74] || [104, 105, 106]
+  var letters1 = []; // ['H', 'I', 'J'] || ['h', 'i', 'j']
+  if (str === str.toUpperCase()) {
```

Comment:
Tu código no funciona si str contiene una combinación de letras mayúsculas y minúsculas (por ejemplo, probar el input 'Hola')

Reply:
Es mejor poner el `if` dentro del for loop, y probar cada letra para ver si es mayúsculas o minúsculas - no probes el string completo